apps/Renoise (Demo)/uninstall

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cd /tmp
+wget "https://raw.githubusercontent.com/Crilum/stuff/main/install.sh" || error "Failed to download 'install.sh'!"
+chmod +x install.sh || error "Failed to make 'install.sh' executable!"
+sudo ./install.sh -u
+if [ "$?" == "1" ]; then
+  status "Uninstall finished!"
+elif [ "$?" == "0" ]; then
+  error "Failed to uninstall Renoise with 'install.sh -u'"
+else
+  warning "Unknown status code... Renoise may or may not have uninstalled successfully."
+fi
```

Comment:
this is unprofessional, why do you feel the need to do this? if Renoise is extracted, the the installs script always remains and you should run it RESOURCES_PATH (aka, `RESOURCES_PATH=$SYSTEM_LOCAL_SHARE/renoise-$RENOISE_VERSION`

you see here this is part of the install:
```
    for f in "install.sh" "uninstall.sh" "Installer"; do
      cp -r $SCRIPT_PATH/$f $RESOURCES_PATH || bail_out
    done
```
please edit your uninstall script to use the locally available one.

Reply:
Thanks, I'll fix this.

Reply:
Does my latest commit do what you were wanting?

Reply:
yup, assuming that is the correct path, that is good

Reply:
Yep, it is. I tested it.